packaging/rpm/rear.spec

```diff
@@ -18,8 +18,7 @@ License: GPL-3.0
 Group: Applications/File
 URL: http://relax-and-recover.org/
 
-# as GitHub stopped with download section we need to go back to Sourceforge for downloads
-Source: https://sourceforge.net/projects/rear/files/rear/%{version}/rear-%{version}.tar.gz
+Source: https://github.com/rear/rear/archive/%{version}.tar.gz#/rear-%{version}.tar.gz
 
 # BuildRoot: is required for SLES 11 and RHEL/CentOS 5 builds on openSUSE Build Service (#2135)
```

Comment:
@jsmeix please remind me - is SLES 11 going to be deprecated in the next release? If so, we could remove this line.

Reply:
SLES11 is no longer officially supported by ReaR upstream
since a long time and since ReaR 2.7 is released even I
do no longer care about SLE11 at ReaR upstream
so feel free to remove such old and obsolete stuff.

Explanation:
During ReaR 2.7 development I only liked to
not knowingly break ReaR 2.7 for SLE11 because
I might need ReaR 2.7 for SLE11 for special SUSE customers
on very specific request and therefore I liked to avoid
that I would have to fix various things at various places
to make ReaR 2.7 usable for a specific use case on SLE11.